src/services/factories/tokenList.ts

```diff
@@ -218,8 +258,20 @@ export function getTokensFactory(factoryParams: {
     // Get token details for each filtered token
     const tokenDetails = await _fetchTokenDetails(networkId, filteredAddressesAndIds, tokensConfig)
 
+    // Sort tokens
+    const tokenList = _sortTokens(networkId, tokenDetails)
```

Comment:
This is misleading, Sort is mutative, so we don't get a new `tokenList` out of that. One might be inclined to treat `tokenDetails` as unchanged after this line, though we don't do it.

Just writing
```js
_sortTokens(networkId, tokenDetails)
```
explicitly shows  that `tokenDetails` was mutated